server/chat-commands/admin.ts

```diff
@@ -420,6 +420,57 @@ export const commands: Chat.ChatCommands = {
 		`If a [highlight] is specified, only highlights them if they have that term on their highlight list.`,
 	],
 
+	changeprivateuhtml: 'sendprivatehtml',
+	sendprivateuhtml: 'sendprivatehtml',
+	sendprivatehtml(target, room, user, connection, cmd) {
+		room = this.requireRoom();
+		this.checkCan('addhtml', null, room);
+
+		const {targetUser, rest} = this.requireUser(target);
+
+		if (targetUser.locked && !this.user.can('lock')) {
+			this.errorReply("This user is currently locked, so you cannot send them private HTML.");
+			return false;
+		}
+
+		if (!(targetUser.id in room.users)) {
+			throw new Chat.ErrorMessage("You cannot send private HTML to users who are not in this room.");
+		}
+
+		let html: string;
+		let messageType: string;
+		let name: string | undefined;
+		const plainHTML = cmd === 'sendprivatehtml';
+		if (plainHTML) {
+			html = rest;
+			messageType = 'html';
+		} else {
+			[name, html] = this.splitOne(rest);
+			if (!name) return this.parse('/help ' + cmd);
+
+			messageType = `uhtml${(cmd === 'changeprivateuhtml' ? 'change' : '')}|${name}`;
+		}
+
+		html = this.checkHTML(html);
+		if (!html) return this.parse('/help ' + cmd);
+
+		html = `${Utils.html`<div style="color:#888;font-size:8pt">[Private from ${user.name}]</div>`}${Chat.collapseLineBreaksHTML(html)}`;
+		if (plainHTML) html = `<div class="infobox">${html}</div>`;
+
+		targetUser.sendTo(room, `|${messageType}|${html}`);
+
+		this.sendReply(`Sent private HTML to ${targetUser.name}.`);
+	},
+	sendprivatehtmlhelp: [
+		`/sendprivatehtml [userid], [html] - Sends [userid] the private [html]. Requires: * # &`,
+	],
+	sendprivateuhtmlhelp: [
+		`/sendprivateuhtml [userid], [name], [html] - Sends [userid] the private [html] that can change. Requires: * # &`,
+	],
+	changeprivateuhtmlhelp: [
+		`/changeprivateuhtml [userid], [name], [html] - Changes the message previously sent with /sendprivateuhtml [userid], [name], [html]. Requires: * # &`,
+	],
```

Comment:
```suggestion
	sendprivatehtmlhelp: [
		`/sendprivatehtml [userid], [html] - Sends [userid] the private [html]. Requires: * # &`,
	],
	sendprivateuhtmlhelp: [
		`/sendprivateuhtml [userid], [name], [html] - Sends [userid] the private [html] that can change. Requires: * # &`,
	],
	changeprivateuhtmlhelp: [
		`/changeprivateuhtml [userid], [name], [html] - Changes the message previously sent with /sendprivateuhtml [userid], [name], [html]. Requires: * # &`,
	],
```

It might be cleaner to compress these into one help under `sendprivatehtml`? I think that's generally what we do elsewhere.